charts/spiderpool/templates/deployment.yaml

```diff
@@ -175,6 +175,8 @@ spec:
           value: {{ .Values.global.cniConfHostPath | quote }}
         - name: SPIDERPOOL_COORDINATOR_ENABLED
           value: {{ .Values.coordinator.enabled | quote }}
+        - name: SPIDERPOOL_COORDINATOR_DEAFULT_NAME
```

Comment:
new environment should be added to document

Reply:
Added.